api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -70,9 +69,52 @@ message ListenersConfigDump {
     api.v2.Listener listener = 2;
 
     // The timestamp when the Listener was last updated.
+    // FIXME: reviewer we may eventually want last updated and last _successfully_ updated.
+    // Given similarity between last_updated (attempt or success) and
+    // last_update should this maybe be last_update_attempt so last_update
+    // matches?
     google.protobuf.Timestamp last_updated = 3;
   }
 
+  // Describes a dynamically loaded listener via the LDS API.
+  // FIXME rename? because otherwise v2->alpha is super confusing but this seems
```

Comment:
Sorry what do you mean here?

Reply:
Sorry it was unclear
Prior to my changes, DynamicListener was a structure which had {list of active, list of warming, list of draining}
With this PR the message DyamicListener is now {?active? ?warming? ?draining? } and we have a list of them.

Basically is it Ok to reuse the name or will it cause more confusion

Reply:
I think it's fine to pick the optimal name for the current structure, given this is still an alpha proto.

Reply:
Cool, that's the feedback I was looking for :-)

OK in general, I moved this to a reusable proto with Any, cleaned up the FIXMES and restructured the rest of the protos.   I think we're in enough agreement on structure I'll start in on code changes tomorrow, but please weigh in with any other API suggestions to minimize that code churn!